index.js

```diff
@@ -1,23 +1,66 @@
 'use strict';
-const {promisify} = require('util');
 const childProcess = require('child_process');
-const neatCsv = require('neat-csv');
-const sec = require('sec');
+const {promisify} = require('util');
+const {pipeline} = require('stream');
+const csv = require('csv');
+const csvHeaders = require('./csv-headers');
+const transform = require('./transform');
 
 const execFile = promisify(childProcess.execFile);
 
-module.exports = async (options = {}) => {
+csv.parse[promisify.custom] = (input, options) => new Promise(resolve => {
+	csv.parse(input, options, (_, records) => resolve(records));
+});
```

Comment:
Why can't you just use `promisify` directly? And if you're going to do this, you might as well just drop using `promisify`.